mapper/transaction.go

```diff
@@ -506,8 +506,8 @@ func parseErc721Txs(transferLog ethtypes.Log, opsLen int64) []*types.Operation {
 	addressTo := transferLog.Topics[2]
 	erc721Index := transferLog.Topics[3] // Erc721 4th topic is the index.  Data is empty
 	metadata := map[string]interface{}{
-		ContractAddressMetadata:  contractAddress.String(),
-		IndexTransferredMetadata: erc721Index.String(),
+		"contractAddress":  contractAddress.String(),
+		"indexTransferred": erc721Index.String(),
```

Comment:
should we change `contractAddress` -> `contract_address` and `indexTransferred` to `index_transferred`? All the other metadata maps use snake case rather than camel case.